sympy/integrals/manualintegrate.py

```diff
@@ -1575,6 +1576,9 @@ def manualintegrate(f, var):
     sympy.integrals.integrals.Integral.doit
     sympy.integrals.integrals.Integral
     """
+    if len(f.args) > 1 :
+        if all(i.has(exp) for i in f.args):
+            f = f.expand()
```

Comment:
`expand` is very general, try to use only the necessary expansion(s).

Reply:
I just change it to cancel and checks failed due to some other reasons.

Reply:
The test failure is unrelated to your change. If you push another change to your code it will rerun the tests and that failure should go away.